.circleci/config.yml

```diff
@@ -26,14 +26,16 @@ executors:
         xpack.license.self_generated.type: trial
         xpack.security.authc.api_key.enabled: true
         ELASTIC_PASSWORD: password
-    - image: docker.elastic.co/enterprise-search/enterprise-search:7.8.0
+        ES_JAVA_OPTS: -Xms512m -Xmx512m
+    - image: docker.elastic.co/enterprise-search/enterprise-search:7.12.0
       name: appsearch
       environment:
         elasticsearch.host: http://elasticsearch:9200
         elasticsearch.password: password
         ENT_SEARCH_DEFAULT_PASSWORD: password
         secret_management.encryption_keys: "[changeme]"
         allow_es_settings_modification: true
+        elasticsearch.startup_retry.interval: 15
```

Comment:
This retry interval is a temporary workaround for a known race condition in Enterprise Search startup when waiting for the ES license server to be available. Without it, App Search can sporadically fail to start.